README.md

```diff
@@ -1,7 +1,26 @@
 
-# penalties
+# Penalties
 
-This is a placeholder README.md for a new repository
+The backend for the penalties services. It handles the following:
```

Comment:
can we specify the 'penalties services' that this penalties serves as a backend for.